tst/EnergyPlus/unit/HVACSizingSimulationManager.unit.cc

```diff
@@ -123,10 +123,10 @@ class HVACSizingSimulationManagerTest : public EnergyPlusFixture
         state->dataGlobal->NumOfTimeStepInHour = 4;
         state->dataWeather->TimeStepFraction = 1.0 / double(state->dataGlobal->NumOfTimeStepInHour);
 
-        state->dataOutputProcessor->TimeValue.at(OutputProcessor::TimeStepType::Zone).TimeStep = &state->dataGlobal->TimeStepZone;
-        state->dataOutputProcessor->TimeValue.at(OutputProcessor::TimeStepType::Zone).CurMinute = 0; // init
-        state->dataOutputProcessor->TimeValue.at(OutputProcessor::TimeStepType::System).TimeStep = &state->dataHVACGlobal->TimeStepSys;
-        state->dataOutputProcessor->TimeValue.at(OutputProcessor::TimeStepType::System).CurMinute = 0;
+        state->dataOutputProcessor->TimeValue[(int)OutputProcessor::TimeStepType::Zone].TimeStep = &state->dataGlobal->TimeStepZone;
+        state->dataOutputProcessor->TimeValue[(int)OutputProcessor::TimeStepType::Zone].CurMinute = 0; // init
+        state->dataOutputProcessor->TimeValue[(int)OutputProcessor::TimeStepType::System].TimeStep = &state->dataHVACGlobal->TimeStepSys;
+        state->dataOutputProcessor->TimeValue[(int)OutputProcessor::TimeStepType::System].CurMinute = 0;
```

Comment:
Interesting that unit tests are manipulating the output variable structs at this level.  Oh well.